includes/ucf-post-list-common.php

```diff
@@ -230,6 +230,44 @@ public static function get_image_or_fallback( $item, $size='large' ) {
 			return $img;
 		}
 
+		/**
+		 * Retrieves the post excerpt and sets a character limit
+		 *
+		 * @author Mike Setzer
+		 * @since 2.0.8
+		 * @param object $item | object containing the WP Post
+		 * @param int $max_words | Maximum characters displayed from excerpt
+		 * @return string $excerpt | Excerpt text
+		 **/
+		public static function get_excerpt ( $item, $max_words) {
```

Comment:
Like Jim noted, this function can be simplified a bit and updated to support existing excerpt filters with something like the following:

```php
public static function get_excerpt( $item, $max_words ) {
	$excerpt = '';

	$custom_filter = function( $l ) use ( $max_words ) {
		if ( $max_words > 0 ) {
				return $max_words;
		}
		return $l;
	};

	add_filter( 'excerpt_length', $custom_filter, 999 );
	$excerpt = wp_strip_all_tags( get_the_excerpt( $item ) );
	remove_filter( 'excerpt_length', $custom_filter, 999 );

	return $excerpt;
}
```

By using `get_the_excerpt()`, we ensure that developer-intended overrides to excerpts throughout the theme are applied (such as trailing ellipses modifications, the baseline excerpt length, etc); `get_the_excerpt()` will also grab a fallback value from the post content if an excerpt isn't explicitly defined.  The `add_filter` and `remove_filter` calls update and then reset the desired excerpt length on the fly.

If shortcode content is still a concern after trying this code, I'd recommend either adding shortcode parsing to excerpts **in your theme** (e.g. via `add_filter( 'the_excerpt', 'do_shortcode' );`), or add a similar function in your theme to strip the shortcode tags entirely; shortcode handling for excerpts probably falls outside of the scope of what the post list shortcode should be configured to do.